languages/zh_hans.php

```diff
@@ -0,0 +1,1616 @@
+<?php
```

Comment:
Translations are managed by Transifex http://learn.elgg.org/en/stable/contribute/i18n.html
Including this would have no effect as the first pull of the translations from Transifex would overrule this file

Reply:
OK, I'll try that. Thank you.